btrack/dataio.py

```diff
@@ -400,7 +404,7 @@ def object_type(self, obj_type: str):
             raise ValueError("Object type must start with ``obj_type_``")
         self._object_type = obj_type
 
-    @property
+    @property  # type: ignore
```

Comment:
What's the reason for all these ignore type comments?

Reply:
It's to do with a long-standing `mypy` bug with two decorators (one being `property`) https://github.com/python/mypy/issues/1362

Reply:
This silences `mypy` moaning